packages/samples/headless-ssr-commerce/app/listing/page.tsx

```diff
@@ -26,15 +26,18 @@ export default async function Listing() {
   const staticState = await listingEngineDefinition.fetchStaticState();
 
   return (
-    <ListingPage
+    <ListingProvider
       staticState={staticState}
       navigatorContext={navigatorContext.marshal}
     >
       <div style={{display: 'flex', flexDirection: 'row'}}>
         <div style={{flex: 1}}>
```

Comment:
Or you could just use flex: 'auto' for all four divs; same comment for the search provider.

Reply:
It makes it so all 4 divs together take 100% of the screen. I like it better how it is now.